tooling/noirc_abi_wasm/build.sh

```diff
@@ -38,7 +38,7 @@ rm -rf $self_path/result >/dev/null 2>&1
 if [ -v out ]; then
   echo "Will install package to $out (defined outside installPhase.sh script)"
 else
-  out="$self_path/outputs/out"
+  export out="$self_path/outputs/out"
```

Comment:
This is needed in general since we don't pass $out to the installPhase script, not entirely sure why it works in the nix environment